content/doc/tutorials/using-jenkinsfile-runner-github-action-to-build-jenkins-pipeline.adoc

```diff
@@ -0,0 +1,378 @@
+                sh 'echo Hello Jenkins!'
+            }
+        }
+    }
+}
+----
+
+=== Create a plugin list file
+
+In this section, you will create a plugin list file, which specifies the plugins you will need to install in the ephemeral Jenkins instance. 
+Please refer to link:https://github.com/jenkinsci/plugin-installation-manager-tool#plugin-input-format[the valid plugin input format]. 
+In our example, we don’t have to install any extra plugins, so you can create an empty plugin list file called “plugins.txt”.
+
+=== Create a workflow definition in the container
+
+In this section, you’ll learn how to create a workflow definition so you can run Jenkins pipeline with GitHub Actions.
+The following steps are required to create the definition.
+
+. In your local GitHub repository, prepare a workflow definition YAML file. 
+For example `ci.yml` under the “.github/workflows” directory.
+. Create the name of your workflow and event that triggers your workflow. 
+Refer to link:https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#available-events[the available events] that trigger workflows. 
+The “push” event triggers the workflow when you push a commit or tag.
++
+[source,yaml]
+----
+name: Java CI
+on: [push]
+----
+. Use an Ubuntu runner for the job.
++
+[source,yaml]
+----
+jobs:
+  job-name:
+    runs-on: ubuntu-latest
+----
+. (Optional) If you want to use `jfr-container-action` later, you need to declare this using the `ghcr.io/jenkinsci/jenkinsfile-runner:master` or any image extending it. 
+If you choose to use `jfr-static-image-action`, you can skip this step.
++
+[source,yaml]
+----
+jobs:
+  job-name:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/jenkinsci/jenkinsfile-runner:master
+----
+. Call the `actions/checkout@v2` to pull your codes into the runner. 
+** “Call” means “uses” in the workflow definition specifically. 
+You can check the details about the “uses” keyword.
++
+[source,yaml]
+----
+- uses: actions/checkout@v2
+----
+. Call the Jenkinsfile-runner actions.
+** If you use `jfr-container-action`, you need to call `jenkinsci/jfr-container-action@master` and provide the necessary inputs.
++
+[source,yaml]
+----
+uses:
+  jenkinsci/jfr-container-action@master
+with:
+  command: run
+  jenkinsfile: Jenkinsfile
+  pluginstxt: plugins.txt
+----
+** If you use `jfr-static-image-action`, you need to call `jenkinsci/jfr-static-image-action@master` and provide the necessary inputs.
++
+[source,yaml]
+----
+uses:
+  jenkinsci/jfr-static-image-action@master
+with:
+  command: run
+  jenkinsfile: Jenkinsfile
+  pluginstxt: plugins.txt
+----
+
+Refer to the full example workflow definition for link:https://jenkinsci.github.io/jfr-action-doc/docs/user-guide/example-workflows.html#container-job-action[jfr-container-action] or link:https://jenkinsci.github.io/jfr-action-doc/docs/user-guide/example-workflows.html#docker-container-action[jfr-static-image-action] to verify your work. 
```

Comment:
It would be good to show what the "complete" example looks like, despite it being on the linked page. This would provide readers with the full picture in addition to the pieces that are explained prior.

Reply:
Great! Let me update it later.

Reply:
Already add it in the latest commit.